javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,102 @@
+// Exercise 1 - FIND THE SHORTEST WORD
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+let shortestWord = "";
+
+function getShortestWord(array) {
+  for (let i = 0; i < array.length; i++) {
+    shortestWord = array[0];
+    if (array[i].length < shortestWord.length) {
+      shortestWord = array[i];
+      return shortestWord;
+    }
+  }
+}
+
+console.log(getShortestWord(danishWords));
+
+// FIND AND COUNT THE DANISH LETTERS
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med rød bær";
+const danishString3 = "";
+
+function findDanishLetters(sentence) {
+  const danishLetters = {};
```

Comment:
This function is a bit to big. What i would recommend you to do instead is to move some of the functionality of this function into a new function. For example you would have a function that counts the letters and so on and another one that controls the logic of setting the danishLetters object.